backend/api/views.py

```diff
@@ -1,8 +1,90 @@
-from django.shortcuts import render
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from .serializers import FiboUserSerializer
+from .models import FiboUser
+from django.contrib.auth import authenticate
 
-# Create your views here.
-from django.http import HttpResponse
 
+@api_view(['GET'])
+def getUsers(request):
+    users = FiboUser.objects.all()
+    serializer = FiboUserSerializer(users, many=True)
+    return Response(serializer.data)
 
-def index(request):
-    return HttpResponse("Hello, world. You're at the api index.")
\ No newline at end of file
+
+@api_view(['GET'])
+def getUser(request, pk):
+    user = FiboUser.objects.get(id=pk)
+    serializer = FiboUserSerializer(user, many=False)
+    return Response(serializer.data)
+
+
+@api_view(['DELETE'])
+def deleteUser(request, pk):
+    user = FiboUser.objects.get(id=pk)
+    user.delete()
+    return Response(f'User with id {pk} was deleted')
+
+
+@api_view(['POST'])
+def registerUser(request):
+    # Note that username and email are the same
+    user = FiboUser.objects.create_user(username=request.data['email'], email=request.data['email'],
+                                        password=request.data['password'])
+    return Response(f'User with id {user.id} and email {user.email} was created')
+
+
+@api_view(['POST'])
+def authenticateUser(request):
+    # Note that username and email are the same
+    user = authenticate(username=request.data['email'], password=request.data['password'])
+    if user is not None:
+        # TODO login(request,user)
+        # redirect to a success page
+        return Response(f'Authenticated user {user.email}')
+    else:
+        return Response('Invalid login')
```

Comment:
I think this should use http status codes for better testability.

If a user could authenticate correctly, it should send 200, otherwise 401 (see https://stackoverflow.com/a/45357230)